### PR TITLE
Update the output of tedge connect in tutorial and how-to guide

### LINKED ISSUE #905: The command outputs of some doc are outdated
**Is your documentation request related to a problem? Please describe.**
Some outputs of tedge commands are outdated. We should keep them updated.

1) https://github.com/thin-edge/thin-edge.io/blob/main/docs/src/tutorials/connect-c8y.md

```
$ sudo tedge connect c8y
Checking if systemd is available.

Checking if configuration for requested bridge already exists.

Validating the bridge certificates.

Saving configuration for requested bridge.

Restarting mosquitto service.

Awaiting mosquitto to start. This may take up to 5 seconds.

Enabling mosquitto service on reboots.

Successfully created bridge connection!

Checking if tedge-mapper is installed.

Starting tedge-mapper-c8y service.

Persisting tedge-mapper-c8y on reboot.

tedge-mapper-c8y service successfully started and enabled!

Sending packets to check connection. This may take up to 10 seconds.

Try 1 / 2: Sending a message to Cumulocity.
Received expected response message, connection check is successful.

Enabling software management.

Checking if tedge-agent is installed.

Starting tedge-agent service.

Persisting tedge-agent on reboot.

tedge-agent service successfully started and enabled!
```

2) https://github.com/thin-edge/thin-edge.io/blob/main/docs/src/howto-guides/004_connect.md

```
$ sudo tedge connect c8y
Checking if systemd is available.

Checking if configuration for requested bridge already exists.

Validating the bridge certificates.

Creating the device in Cumulocity cloud.

Saving configuration for requested bridge.

Restarting mosquitto service.

Awaiting mosquitto to start. This may take up to 5 seconds.

Enabling mosquitto service on reboots.

Successfully created bridge connection!

Sending packets to check connection. This may take up to 2 seconds.

Connection check is successfull.

Checking if tedge-mapper is installed.

Starting tedge-mapper-c8y service.

Persisting tedge-mapper-c8y on reboot.

tedge-mapper-c8y service successfully started and enabled!

Enabling software management.

Checking if tedge-agent is installed.

Starting tedge-agent service.

Persisting tedge-agent on reboot.

tedge-agent service successfully started and enabled!
```

and

```
$ sudo tedge connect c8y
Checking if systemd is available.

Checking if configuration for requested bridge already exists.

Validating the bridge certificates.

Saving configuration for requested bridge.

Restarting mosquitto service.

Awaiting mosquitto to start. This may take up to 5 seconds.

Persisting mosquitto on reboot.

Successfully created bridge connection!

Checking if tedge-mapper is installed.

Starting tedge-mapper service.

Persisting tedge-mapper on reboot.

tedge-mapper service successfully started and enabled!

Sending packets to check connection. This may take up to 10 seconds.

Try 1 / 2: Sending a message to Cumulocity. ... No response. If the device is new, it's normal to get no response in the first try.
Try 2 / 2: Sending a message to Cumulocity. ... No response.
Warning: Bridge has been configured, but Cumulocity connection check failed.
```

3) https://github.com/thin-edge/thin-edge.io/blob/main/docs/src/tutorials/connect-azure.md

```
$ sudo tedge connect az

Checking if systemd is available.

Checking if configuration for requested bridge already exists.

Validating the bridge certificates.

Saving configuration for requested bridge.

Restarting mosquitto service.

Awaiting mosquitto to start. This may take up to 5 seconds.

Persisting mosquitto on reboot.

Successfully created bridge connection!

Checking if tedge-mapper is installed.

Starting tedge-mapper service.

Persisting tedge-mapper on reboot.

tedge-mapper service successfully started and enabled!

Sending packets to check connection. This may take up to 10 seconds.

Received expected response message, connection check is successful.
```

### PR DESCRIPTION
**Pull request by rina23q**
## Proposed changes

We changed the output of `tedge connect c8y` and `tedge connect az`, however, the sample output in our doc were still using the old output. This PR updates such docs.

Plus, to change all `c8y` or `az` output to `Cumulocity` or `Azure` by `tedge disconnect`. This keeps consistency with the output of `tedge connect`. 

## Types of changes

What types of changes does your code introduce to `thin-edge.io`?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue

Close #905.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [x] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [x] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/src/howto-guides/004_connect.md [hunk 1]
```diff
@@ -113,27 +113,32 @@ To remove existing connection and create new one follow the advice and issue [`t
 
 ```shell
 $ sudo tedge disconnect c8y
-Removing Cumulocity bridge.
+Removing C8y bridge.
```
Comment (didier-wenzek) [inline docs/src/howto-guides/004_connect.md:116]: Why this change? `Cumulocity` sounds as the correct name and `C8y` is an acronym far less well-known than `k8s` ;-)

Comment (rina23q) [reply docs/src/howto-guides/004_connect.md:116]: Because of this commit last year. https://github.com/thin-edge/thin-edge.io/commit/81526b08d56d12d33ee98ee5c4911bc02a398eda

In `disconnect_bridge.rs`, the current code is
```
println!("Removing {:?} bridge.\n", self.cloud);
```

But if we change it to the below, it will be back to `Cumulocity`, not `c8y`. 
```
        let cloud_name: String = self.cloud.into();
        println!("Removing {} bridge.\n", cloud_name);
```

Comment (rina23q) [reply docs/src/howto-guides/004_connect.md:116]: Should I add the fix to this PR?

**Review by initard — APPROVED** [review]
